maps/sccv_horizon/sccv_horizon-4_centcomm.dmm

```diff
@@ -20952,17 +20952,15 @@
 /turf/unsimulated/floor/plating,
 /area/centcom/legion/hangar5)
 "baV" = (
-/obj/effect/decal/fake_object{
-	density = 1;
-	desc = "Ding.";
-	icon = 'icons/obj/doors/doorlift.dmi';
-	icon_state = "door_closed";
-	name = "elevator door";
-	opacity = 1
-	},
-/turf/unsimulated/floor{
-	icon_state = "cult";
-	name = "plating"
+/turf/unsimulated/wall/fakeairlock{
+	icon = 'icons/obj/doors/basic/single/external/door.dmi';
+	icon_state = "door_locked";
+	name = "external airlock";
+	opacity = 0;
+	permit_ao = 0;
+	dir = 2;
+	pixel_x = -16;
+	pixel_y = -16
```

Comment:
This looks absolutely atrocious, there has to be a better and sensibly doable way to do this

Reply:
as below